code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

```diff
@@ -30,34 +31,41 @@
 	blood_volume =	0
 	min_age =		18
 	max_age =		200
-	brute_mod =		0.2 //Brute isn't very effective, they're made of dust
-	burn_mod =		2.0 //Burn, however, is
+	brute_mod =		0.40 //Brute isn't very effective, they're made of dust
+	burn_mod =		1.2 //Burn, is more effective due to grahpine conduits. Hard to move to fill missing areas.
 	oxy_mod =		0
 
 	cold_level_1 = 280 //Default 260 - Lower is better
 	cold_level_2 = 220 //Default 200
 	cold_level_3 = 130 //Default 120
 
-	heat_level_1 = 320 //Default 360
-	heat_level_2 = 370 //Default 400
-	heat_level_3 = 600 //Default 1000
+	heat_level_1 = 552 //Default 360
+	heat_level_2 = 553 //Default 400
+	heat_level_3 = 570 //Default 1000
+	heat_discomfort_level = 552
+
+	//melting point of Fullerene is 553K. Proteans take minor damage slightly below it. They start taking massive damage VERY fast above it,
+	//Proteans start melting literally >553K on the inside
+	//Graphine has an insane melting point but there's nothing holding them in place.
+
 
 	//Space doesn't bother them
 	hazard_low_pressure = -1
-	hazard_high_pressure = 200 //They can cope with slightly higher pressure
+	hazard_high_pressure = 1000 //Proteans laugh at high pressures other synths cant. Fullerene is highly fluid and graphine would shift along with it.
+	//I would code in something like pressure causing slowdown.
 
-	//Cold/heat does affect them, but it's done in special ways below
+	//Cold does  affect them, but it's done in special ways below
 	cold_level_1 = -INFINITY
 	cold_level_2 = -INFINITY
 	cold_level_3 = -INFINITY
-	heat_level_1 = INFINITY
-	heat_level_2 = INFINITY
-	heat_level_3 = INFINITY
 
-	body_temperature =      290
 
-	siemens_coefficient =   3 //Very bad zappy times
-	rarity_value =          5
+	body_temperature =      283 //10C
+
+	siemens_coefficient =   1.5 //Very bad zappy times, balance from 2.0 to insta perma lock. This is to compensate for the fact we can the theroetical graphite circuits are embedded very close on the surface of the body and everywhere.
```

Comment:
This is a PRETTY insane buff over what they originally were.

Reply:
The Siemens Coeff I'm warry. I don't want an instant 25 second stunlock like before but at the same time I don't want them to be human levels of shock resistant.
Also since we cant critical hit balance somehow??

Also. I don't know about you. But a protean can die from touching a vending machine on just the main SMES alone. I've had it happen multiple times. Keep in mind proteans CANNOT be revived as of now.